src/main/java/org/sefglobal/scholarx/controller/ProgramController.java

```diff
@@ -53,4 +56,38 @@ public Mentor applyAsMentor(@PathVariable long id,
         long profileId = 1; // TODO: Get the profileId from headers
         return programService.applyAsMentor(id, profileId, mentor);
     }
+
+    @GetMapping("/{id}/mentor")
+    @ResponseStatus(HttpStatus.OK)
+    public Mentor getLoggedInMentor(@PathVariable long id) throws ResourceNotFoundException {
+        long profileId = 1;  // TODO: Get the profileId from headers
+        return programService.getLoggedInMentor(id, profileId);
+    }
+
+    @GetMapping("/{id}/mentees")
```

Comment:
Path of this API is `mentor/{id}/mentees`

Reply:
So we can just find the mentor object by the `id` taken from the path parameter. And the `profileId` taken from headers is only used to validate the mentor. Am i right?

Reply:
No need of a `profileId`. Since the API is getting the `mentorId` it self, just validate if mentor is in the correct state as stated in here: https://github.com/sef-global/scholarx/pull/47/files#r513133892

Reply:
Done